### PR TITLE
FISH-6436 : keystores change documentation

### PR DESCRIPTION
**Pull request by luiseufrasio**

### CODE REVIEW

File: community/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates.adoc [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
 The Payara Server uses a dedicated key and certificate store when using SSL/TLS connections. The files are located in the configuration directory of the domain `<payara-home>/glassfish/domains/<domain-name>/config` and referenced through JVM options so that they are used for the entire JVM.
 
-_cacerts.jks_: Java Keystore formatted file used as Trust Store file holding the X.509 certificates.
-_keystore.jks_: Java Keystore formatted file containing the Cryptographic keys, like the private keys associated with the certificates.
+_cacerts.p12_: Java Keystore formatted file used as Trust Store file holding the X.509 certificates.
+_keystore.p12_: Java Keystore formatted file containing the Cryptographic keys, like the private keys associated with the certificates.
 
 These keystores are protected with the Master Password of your domain.
 
+IMPORTANT: From Payara version 6 the keystores were migrated from JKS to PKCS12 format.
```
Comment (JamesHillyard) [inline community/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates.adoc:11]: This isn't strictly true as 6.2022.1, 6.2022.2 and 6.2023.1 use JKS, should instead say 6.2023.2 and later

File: enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates.adoc [hunk 2]
```diff
@@ -3,11 +3,13 @@
...
 Payara Server uses a dedicated key and certificate store when using SSL/TLS connections. The files are located in the configuration directory of the domain `<payara-home>/glassfish/domains/<domain-name>/config` and referenced through JVM options so that they are used for the entire JVM.
 
-`keystore.jks`:: Java Keystore formatted file containing the Cryptographic keys, like the private keys associated with the certificates.
-`cacerts.jks`:: Java Keystore formatted file used as Trust Store file holding the X.509 certificates.
+`keystore.p12`:: Java Keystore formatted file containing the Cryptographic keys, like the private keys associated with the certificates.
+`cacerts.p12`:: Java Keystore formatted file used as Trust Store file holding the X.509 certificates.
 
 These keystores are protected with the Master Password of your domain.
 
+IMPORTANT: From Payara version 6 the keystores were migrated from JKS to PKCS12 format.
```
Comment (JamesHillyard) [inline enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates.adoc:11]: See other comment

**Review by JamesHillyard — CHANGES_REQUESTED** [review]

File: community/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates.adoc (hunk 1 above)
Comment (luiseufrasio) [reply community/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates.adoc:11]: done

File: enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates.adoc (hunk 2 above)
Comment (luiseufrasio) [reply enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates.adoc:11]: done

**Review by JamesHillyard — APPROVED** [review]
Looks good, please don't merge this until documentation cuts have been taken for the current release which doesn't include this change

**Pull request merged**